modules/manage/partials/kubernetes/reset-object-storage-properties.adoc

```diff
@@ -0,0 +1,2 @@
+CAUTION: Do not set an object storage property to an empty string (`""`) or `null` to reset it. This may result in invalid or incomplete configuration.
```

Comment:
oooooo, so this is tricky. _By default_ unsetting a key will NOT unset the cluster config value as that's been the historical behavior for quite a while now. We recently [added an annotation](https://github.com/redpanda-data/cloudv2/pull/22464/files#diff-cc6414e0662bb809d61129b55812a4dd977bdd246c1206a3f8dcfb2232cc6590R9) that will opt you into being fully declarative.

I think we should update this to point users at `rpk cluster config set` (though I'm not sure if that can actually unset a value 🤔 ) or denote the annotation.

Reply:
Yep, LGTM!